src/Test/Tasty/Bench.hs

```diff
@@ -617,6 +625,7 @@ module Test.Tasty.Bench
   , nfAppIO
   , whnfAppIO
   , measureCpuTime
+  , measureWallClockTime
```

Comment:
I'm not convinced that we need feature parity here, let's not expose `measureWallClockTime` yet.

Reply:
Sure, I just added it for consistency, but I agree that it is not necessary at all.